doc/changelog/08-vernac-commands-and-options/17489-deprecations.rst

```diff
@@ -0,0 +1,9 @@
+- **Changed:**
+  The names of deprecation warnings now depends on the version
+  in which they were introduced (their "since" field).
+  This enables selectively enabling/disabling/make as error
+  deprecation warnings according to the version number
+  provided in the :attr:`deprecated` attribute
```

Comment:
```suggestion
  The names of deprecation warnings now depend on the version
  in which they were introduced, using their "since" field.
  This enables deprecation warnings to be selectively enabled,
  disabled, or treated as an error, according to the version number
  provided in the :attr:`deprecated` attribute
```

Reply:
Thanks, applied.